flocker/node/agents/cinder.py

```diff
@@ -249,6 +249,22 @@ def __init__(self,
         self.nova_server_manager = nova_server_manager
         self.cluster_id = cluster_id
 
+    def allocation_unit(self):
+        """
+        Return a 1GiB fixed allocation_unit for now; one which should
+        work on all currently supported and soon-to-be supported
+        platforms.
+
+        This is the minimum allocation unit described by the OpenStack
+        Cinder API documentation.
+         * http://developer.openstack.org/api-ref-blockstorage-v2.html#createVolume # noqa
+
+        XXX: This should be loaded from configuration because
```

Comment:
1. XXX needs addressing, at minimum by filing follow up issue.
2. Why would that violate API?